specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,340 @@
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Appeal not found"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "Appeal": {
+        "required": [
+          "penaltyIdentifier",
+          "reasons"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "ID of appeal from mongoDB"
+          },
+          "createdAt": {
+            "description": "Timestamp of appeals creation",
+            "type": "string",
+            "format": "date",
+            "example": "2021-02-01"
+          },
+          "createdBy": {
+            "$ref": "#/components/schemas/CreatedBy"
+          },
+          "penaltyIdentifier": {
+            "$ref": "#/components/schemas/PenaltyIdentifier"
+          },
+          "reasons": {
+            "$ref": "#/components/schemas/Reason"
+          }
+        }
+      },
+      "Attachment": {
+        "required": [
+          "contentType",
+          "id",
+          "name",
+          "size"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "ID of attachement"
+          },
+          "name": {
+            "type": "string",
+            "description": "name of the attachement file"
+          },
+          "contentType": {
+            "type": "string"
+          },
+          "size": {
+            "description": "file size of attachement",
+            "minimum": 1,
+            "type": "integer",
+            "format": "int32"
+          },
+          "url": {
+            "type": "string",
+            "description": "url path to attachment"
+          }
+        }
+      },
+      "CreatedBy": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "id of user creating appeal"
+          },
+          "emailAddress": {
+            "type": "string",
+            "description": "email address of user creating appeal"
+          }
+        }
+      },
+      "OtherReason": {
+        "required": [
+          "description",
+          "title"
+        ],
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string",
+            "description": "title of reason"
+          },
+          "description": {
+            "type": "string",
+            "description": "description of other reason"
+          },
+          "attachments": {
+            "type": "array",
+            "description": "supporting attachments for other reason",
+            "items": {
+              "$ref": "#/components/schemas/Attachment"
+            }
+          }
+        }
+      },
+      "IllnessReason": {
+        "required": [
+          "illPerson",
```

Comment:
I think it reads better removing the `illness` and `ill` prefix from these fields as the top level field describes the type of reason e.g.

```
"reasons": {
    "illness": {
        "person": "",
        "startDate": "",
        "endDate": "",
        ...
    }
}